File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        
        //vector<int> ans;
     int k = 0;
        for( int i=0; i<nums.size(); i++ ) {
            if( nums[i] != val ) {
                nums[k] = nums[i];
                k++;
            }
        }
        return k;
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        
        int n = nums.length;
        int []ans = new int[2*n];
        
        int index = 0;
        for( int i=0; i<nums.length; i++ ) {
            ans[index++] = nums[i];
        }
        index = n;
        for( int i=0; i<nums.length; i++ ) {
            ans[index++] = nums[i];
        }

        return ans;

    }
}